docs/01_about.md

```diff
@@ -44,11 +44,11 @@ MSS is a client/server application written in the language **Python**.
 This is used to create flight plans and discuss these with pilots of
 research aircraft. The flight path is designed by local and remote
 scientists based on meteorologic forecast data on the QT5 MSS gui
-application. A [publication by Rautenhaus et
-al.](http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf)
-summarizes the principle of this software. At the EuroPython conference,
-a [overview
-talk](https://pyvideo.org/europython-2017/mss-software-for-planning-research-aircraft-missions.html)
+application.
+Two publications are available, one by [Bauer et al.](https://gmd.copernicus.org/articles/15/8983/2022/gmd-15-8983-2022.pdf) 
+and the other by [Rautenhaus et al](http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf),
```

Comment:
```suggestion
and the other by [Rautenhaus et al.](http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf),
```
Since "al" is an abbreviation.